### DISCUSSION TITLE
Shared doc to new doc transition

### DISCUSSION THREAD

**Discussion by wbconnor** [davidmyersdev/octo#126] [Ideas]
I really like the shared doc to new doc transition when you start editing a shared doc. It confused me for just a moment, and then I looked in the URL and saw what happened. Might be nice to make a UI signal to the user to let them know. They might think they're editing the original. Also I was thinking it might be cool if there was a shared docs area with a list of docs shared to me. Default sort should be recent, of course.

**Comment by davidmyersdev** [comment]
I love the idea of a shared document list, but there's nothing stored in Firebase (yet) to track which shared documents you've opened. The URL change sounds like a bug rather than a feature, so I will need to test that out to see what happens. 😅
